### ISSUE TITLE
Loss Functions for Different Training Stages

### ISSUE THREAD

**Issue by maomaoeatfish** [Doby-Xu/WithAnyone#17]
This is an excellent piece of work. I would like to ask: are the loss functions the same across all training stages of withanyone (as shown in Equation 8 in the paper)? In addition, does the backbone for fixed-prompt pre-training in Stage 1 refer to the adapter? Thank you very much for your reply.
